spec/features/users_spec.rb

```diff
@@ -1,90 +1,72 @@
-feature "User management" do
+feature "user management" do
   let(:users_index_path) { polymorphic_path([:users]) }
 
-  context "when an admin user visits the users index" do
-    let!(:admin_user) { login_new_admin_user }
-    subject { admin_user }
+  context "when an admin visits the users index" do
+    let!(:admin) { login_new_admin_user }
+    let!(:users) { create_list(:user, 2) }
+    let!(:unapproved_user) { create(:user, approval_at: nil)}
 
     before { visit users_index_path }
 
     it "displays a table of users" do
       expect(page).to have_table("users_table")
     end
 
-    it "fills the table with the right headers and rows" do
-      check_headers_and_values_on_generic_index_page(
-        "users_table", {
-          "Name" => "Example User",
-          "Created At" => subject.created_at_string,
-          "Approval At" => subject.approval_at_string,
-          "Admin" => "true",
-          "Actions" => "Remove Approval Delete",
-        }
-      )
+    it "sees a table with proper headers and rows" do
+      expect(page).to have_css("#user_#{admin.id}", :text => "#{admin.name}")
+      expect(page).to have_css("#user_#{admin.id}", :text => "#{admin.created_at_string}")
+      expect(page).to have_css("#user_#{admin.id}", :text => "#{admin.approval_at_string}")
+      expect(page).to have_css("#user_#{admin.id}", :text => "Delete")
+      expect(page).to have_css("#user_#{admin.id}", :text => "Yes")
+      expect(page).to_not have_css("#user_#{admin.id}", :text => "No")
```

Comment:
If you decide to change the format of the dates, then that'll probably take place inside of the model methods, as it is right now (e.g. User.approval_at_string). The test is matching what's on the DOM with the logic in the model. I think it's sound unless you're manipulating the date format somewhere else.